### PR TITLE
code layout-stars, change demo and test report link

### PR DESCRIPTION
**Pull request by entodasha**
- [x]Yellow stars are added with container modifier + pseudo-selector (NO extra classes)
 - [x]Each BEM block has its own separate file
 - [x]All Typical Mistakes from BEM lesson theory are checked.
 - [x]Code follows all the [Code Style Rules ❗️](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)

### CODE REVIEW

**Comment by entodasha** [comment]
- [x]Yellow stars are added with container modifier + pseudo-selector (NO extra classes)
 - [x]Each BEM block has its own separate file
 - [x]All Typical Mistakes from BEM lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)

File: readme.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ Hold `Alt` key (`Option` on MacOS) to measure distances in Figma.
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://entodasha.github.io/layout_stars/)
+- [TEST REPORT LINK](https://entodasha.github.io/layout_stars/report/html_report/)
```
Comment (volodymyr-soltys97) [inline readme.md:26]: Add these links to the PR description 
image

File: src/styles/stars.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(/src/images/star-active.svg);
+}
```
Comment (volodymyr-soltys97) [inline src/styles/stars.css:32]: These selectors must be combined with a comma.

File: src/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+/* add styles here */
```
Comment (volodymyr-soltys97) [inline src/styles/style.css:1]:
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, your demo link isn't working, you need to make a deploy and check it after on the demo 
image

File: src/index.html [hunk 4]
```diff
@@ -9,10 +9,51 @@
...
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1">
```
Comment (volodymyr-soltys97) [inline src/index.html:23]: Add the blank line between the blocks
```suggestion
    </div>
    
    <div class="stars stars--1">
```

**Review by etojeDenys — APPROVED** [review]
lgtm

**Pull request closed without merging**